tests/queries/0_stateless/3008_amazon_ion_output_format.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-parallel
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+$CLICKHOUSE_CLIENT --query="DROP TABLE IF EXISTS ion";
+
+$CLICKHOUSE_CLIENT --query="CREATE TABLE ion (uint8 UInt8, uint16 UInt16, uint32 UInt32, uint64 UInt64, int8 Int8, int16 Int16, int32 Int32, int64 Int64, float Float32, double Float64, string String, date Date, datetime DateTime, datetime64 DateTime64, array Array(UInt32)) ENGINE = Memory";
```

Comment:
Not all types are tested. There is no `Decimal` types, big integers, `FixedString`, `UUID`, `Map`, `Tuple`, `Date32`, nested data types (like `Array(Map(...))`, etc. Also handling not supported types is not checked.
Also we need to test binary output mode (maybe some tool will be needed to check that binary data is correct, maybe some python code or anything else).